peekingduck/pipeline/nodes/draw/poses.py

```diff
@@ -44,14 +47,95 @@ class Node(AbstractNode):
         |none_output_data|
 
     Configs:
-        None.
+        keypoint_dot_color (:obj:`Union[List[int], str]`): **default = "tomato"** |br|
+            Color of the keypoints should either be a string in :ref:`color-palette`,
+            or a list of BGR values.
+
+        keypoint_connect_color (:obj:`Union[List[int], str]`): **default = "champagne"** |br|
+            Color of the keypoints should either be a string in :ref:`color-palette`,
+            or a list of BGR values.
+
+        keypoint_dot_radius (:obj:`int`): **default = 5** |br|
+            Radius of the keypoints.
+
+    **Color Palette**
+
+    :ref:`color-palette` offers a wide range of default colors for the user to choose from.
+
+    .. _color-palette:
+
+    .. list-table:: PeekingDuck's Color Palette
+       :widths: 20 20
+       :header-rows: 1
+
+       * - Color Palette by Type
+         - Color Palette by Name
+       * - .. figure:: ../assets/api/color_map_by_type.png
+
+           PeekingDuck's Color Palette Sorted by Color Types [1]_.
+
+         - .. figure:: ../assets/api/color_map_by_name.png
+
+           PeekingDuck's Color Palette Sorted by Color Names [1]_.
+
+    .. rubric:: Footnotes
+
+    .. [1] Colors with asterisk indicates PeekingDuck's in-house colors.
     """
 
     def __init__(self, config: Dict[str, Any] = None, **kwargs: Any) -> None:
         super().__init__(config, node_path=__name__, **kwargs)
-        self.keypoint_dot_color = tuple(self.config["keypoint_dot_color"])
-        self.keypoint_connect_color = tuple(self.config["keypoint_connect_color"])
-        self.keypoint_text_color = tuple(self.config["keypoint_text_color"])
+        self._validate_configs()
+
+        self.keypoint_dot_color = get_color(self.config["keypoint_dot_color"])
+        self.keypoint_connect_color = get_color(self.config["keypoint_connect_color"])
+
```

Comment:
For the time being I revert back to `self.config["keypoint_dot_color"]` as `mypy` throwing `has-type` error:

```
Cannot determine type of "keypoint_dot_color"
```

Reply:
Just another thought, is there a need to reassign `self.keypoint_dot_color = get_color(...)` instead of just passing it directly to `Pose`'s constructor? Eg:

```
self.pose = Pose(
    get_color(self.keypoint_dot_color),
    ...
)
```

Reply:
I think it is possible, I can change it accordingly.